application/helpers/admin/import_helper.php

```diff
@@ -1408,6 +1408,15 @@
             if (isset($insertdata['surveyls_email_confirm'])) {
                 $insertdata['surveyls_email_confirm'] = translateLinks('survey', $iOldSID, $iNewSID, $insertdata['surveyls_email_confirm']);
             }
+            if (isset($insertdata['attachments'])) {
+                $attachments = unserialize($insertdata['attachments']);
```

Comment:
Eh, this one?

Reply:
I think it is something which we can't handle here.
`attachments` store a serialization of the email templates.

So, if we want to treat it differently, we should store it differently I think, right?

Reply:
Let's ping @c-schmitz

Reply:
> Let's ping @c-schmitz

The Carsten-Signal ?? :)

Reply:
Yes, we should never use serialization for user controlled data.
Rather convert it to JSON in a database upgrade, save it as JSON in the future.
and ignore on import if the user tries to import serialized data.

Reply:
>  and ignore on import if the user tries to import serialized data.

Maybe we need a "lss" history version ? Because this mean lss with attributes exported now don'st work at version X :)

Reply:
So, this ticket would be on hold until the new way of storing data would be set?

Reply:
> So, this ticket would be on hold until the new way of storing data would be set?

Good point …